keps/sig-auth/1205-bound-service-account-tokens/README.md

```diff
@@ -482,6 +485,78 @@ are properly reloading tokens by:
   - upgrade test:
     test/e2e/upgrades/serviceaccount_admission_controller_migration.go
 
+### Rollout, Upgrade and Rollback Planning
+
+_This section must be completed when targeting beta graduation to a release._
+
+- **How can a rollout fail? Can it impact already running workloads?**
+  Try to be as paranoid as possible - e.g., what if some components will restart
+  mid-rollout?
+
+- **What specific metrics should inform a rollback?**
+
+- **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
+  Describe manual testing that was done and the outcomes.
+  Longer term, we may want to require automated upgrade/rollback tests, but we
+  are missing a bunch of machinery and tooling and can't do that now.
+
+- **Is the rollout accompanied by any deprecations and/or removals of features, APIs,
+  fields of API types, flags, etc.?**
+  Even if applying deprecation policies, they may still surprise some users.
+
+### Monitoring Requirements
+
+_This section must be completed when targeting beta graduation to a release._
+
+- **How can an operator determine if the feature is in use by workloads?**
```

Comment:
describe these metrics:

* serviceaccount_valid_tokens_total
* serviceaccount_legacy_tokens_total
* apiserver_request_total and apiserver_request_duration_seconds metrics with `group="",version="v1",resource="serviceaccounts",subresource="token"` labels

Reply:
done